colin/core/checks/abstract_check.py

```diff
@@ -16,16 +16,42 @@
 import json
 
 from six import iteritems
+from colin.core.fmf_metadata_loader import get_fmf_from_class, set_fmf_metadata
 
 
 class AbstractCheck(object):
     name = None
+    message = ""
+    description = ""
+    reference_url = ""
+    tags = []
+    base_init_list = ["message", "description", "reference_url", "tags"]
```

Comment:
so why not just do `__init__(self, message, description, reference_url, tags)`?

Reply:
because this allows me, pass init parameters and assign them as instance variables from fmf data.